graylog2-server/src/main/java/org/graylog2/plugin/streams/Stream.java

```diff
@@ -50,8 +50,6 @@ public static MatchingType valueOfOrDefault(String name) {
 
     String getContentPack();
 
-    Collection<AlertCondition> getAlertConditions();
```

Comment:
This would break existing plugins that are working with Graylog 2.0.x and which are using this method. Any reason to remove it and not simply mark it as deprecated (to remove it in Graylog 3.x)?